packages/api/orm/migration/1638312144983-initial.ts

```diff
@@ -0,0 +1,232 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class initial1638312144983 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const sql = `    
+    SET statement_timeout = 0;
+    SET lock_timeout = 0;
+    SET idle_in_transaction_session_timeout = 0;
+    SET client_encoding = 'UTF8';
+    SET standard_conforming_strings = on;
+    SELECT pg_catalog.set_config('search_path', '', false);
+    SET check_function_bodies = false;
+    SET xmloption = content;
+    SET client_min_messages = warning;
+    SET row_security = off;
+    
+    CREATE EXTENSION IF NOT EXISTS "uuid-ossp" WITH SCHEMA public;
+    
+    COMMENT ON EXTENSION "uuid-ossp" IS 'generate universally unique identifiers (UUIDs)';
+    
+    CREATE TYPE public.application_provisioningstatus_enum AS ENUM (
+        'not_started',
+        'in_progress',
+        'failed',
+        'complete'
+    );
+    
+    
+    ALTER TYPE public.application_provisioningstatus_enum OWNER TO postgres;
+    
+    CREATE TYPE public.application_status_enum AS ENUM (
+        'not_started',
+        'in_progress',
+        'failed',
+        'complete'
+    );
+    
+    ALTER TYPE public.application_status_enum OWNER TO postgres;
+    
+    CREATE TYPE public.environment_provisioningstatus_enum AS ENUM (
+        'not_started',
+        'in_progress',
+        'failed',
+        'complete'
+    );
+    
+    ALTER TYPE public.environment_provisioningstatus_enum OWNER TO postgres;
+    
+    CREATE TYPE public.environment_status_enum AS ENUM (
+        'not_started',
+        'in_progress',
+        'failed',
+        'complete'
+    );
+    
+    ALTER TYPE public.environment_status_enum OWNER TO postgres;
+    
+    CREATE TYPE public.portfolio_csp_enum AS ENUM (
+        'CSP A',
+        'CSP B'
+    );
+    
+    ALTER TYPE public.portfolio_csp_enum OWNER TO postgres;
+    
+    CREATE TYPE public.portfolio_provisioningstatus_enum AS ENUM (
+        'not_started',
+        'in_progress',
+        'failed',
+        'complete'
+    );
+    
+    ALTER TYPE public.portfolio_provisioningstatus_enum OWNER TO postgres;
+    
+    CREATE TYPE public.portfolio_status_enum AS ENUM (
+        'not_started',
+        'in_progress',
+        'failed',
+        'complete'
+    );
+    
+    ALTER TYPE public.portfolio_status_enum OWNER TO postgres;
+    
+    CREATE TYPE public.task_order_filescanstatus_enum AS ENUM (
+        'pending',
+        'accepted',
+        'rejected'
+    );
+    
+    ALTER TYPE public.task_order_filescanstatus_enum OWNER TO postgres;
+    
+    SET default_tablespace = '';
+    SET default_table_access_method = heap;
+    
+    CREATE TABLE public.application (
+        id uuid DEFAULT public.uuid_generate_v4() NOT NULL,
+        "createdAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "updatedAt" timestamp without time zone DEFAULT now() NOT NULL,
```

Comment:
These times are always going to be UTC per the DB config I believe. I don't even know if you can configure that in the RDS Parameter Group. Is that okay?

Reply:
Working on the initial models led me to believe TZ would come from DB config.

Reply:
After discussion, expecting UTC coming from DB config.